setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name='nimiqclient',
+    version='0.0.7',
+    description='A python client for the Nimiq JSON-RPC API',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='http://github.com/rraallvv/NimiqClientPython',
+    author='Nimiq Community',
+    author_email='info@nimiq.com',
+    license='MIT',
```

Comment:
We use the Apache license. :)

Reply:
Fixed here https://github.com/rraallvv/python-client/blob/master/setup.py#L15